### ISSUE TITLE
Voice commands not detected

### ISSUE THREAD

**Issue by isidorn** [b4rtaz/voice-assistant#3]
I have the server nicely running on my windows VM, and I have the extension installed with my VS Code insiders.
However none of the voice commands I say are being recognised. I was loading the default commands and was trying something like "do copy", "do paste"

Here's the output from the server.
The only success I had was with "do undo"

![Screenshot 2021-05-28 at 10 25 25](https://user-images.githubusercontent.com/1926584/119954191-09336300-bf9f-11eb-9475-12017faa1f71.png)

**Comment by b4rtaz** [comment]
The server is based on [Windows Desktop Speech](https://www.nuget.org/packages/System.Speech/5.0.0) and it's imperfect. In my case I did two thinks to get better results:

1. I've boosted the power of my microphone in Windows settings.
![Microphone boost](https://user-images.githubusercontent.com/12797776/119955098-fe2d0280-bf9f-11eb-9e00-21bad3120e42.png)  
  I recommend to [test your microphone](https://www.onlinemictest.com/) before. The microphone signal should be strong.  
  ![Microphone waves](https://user-images.githubusercontent.com/12797776/119955718-9aefa000-bfa0-11eb-8882-e0912bca629f.png)

2. I'm Polish and my accent isn't perfect. So it took some my time to pronounce correctly and sharp phrases.
